app/components/MineralCard/Discovery.tsx

```
@@ -54,16 +54,18 @@ export default function DiscoverySnippet({ isGrouping, slug, data } : { isGroupi
                      </div>
+                     <hr/>
                      <div className="w-auto max-h-[20vh] overflow-auto">
-                       <ul className="flex flex-col space-y-1 list-decimal list-inside marker:text-slate-500 marker:font-normal">
+                       <ul
+                         className="flex flex-col space-y-1 list-decimal list-inside marker:text-slate-500 marker:font-normal">
                          {countryData.map((item_, i) => {
                            return (
                              <li key={i} className="flex-wrap">
                                <span className="font-medium flex-1">{item_.name}</span>
-                               {item_.formula && (<span className="font-normal ml-2 float-right" dangerouslySetInnerHTML={{ __html: item_.formula }}></span>)}
+                               {item_.formula && (<span className="font-normal ml-2 float-right" dangerouslySetInnerHTML={{__html: item_.formula}}></span>)}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)